package fr.dawan.exception_;

public class UtilisateurTest {

	public static void main(String[] args) {
		
		int nbre = 7;
		
		/*
		 * Le bloc try-catch permet de traiter les exceptions 
		 * le bloc try permet de definir un ensemble d'instructions
		 * à surveiller (ex :nbre / 0) susception de lever une 
		 * exception. 
		 * Le bloc Catch va intercepter les exceptions  générées par 
		 * les instructions du bloc try 
		 */
		try {
			System.out.println(nbre / 0);
		} catch(Exception e) {
			System.out.println(e.getMessage());
		}
		
		System.out.println("Continue...");
		 
	}

}
